MemoryMgr.h

```diff
@@ -6,14 +6,15 @@
 
 #define WRAPPER __declspec(naked)
 #define DEPRECATED __declspec(deprecated)
-#define WRAPARG(a) ((int)a)
 
 #ifdef _MSC_VER
 #define EAXJMP(a) { _asm mov eax, a _asm jmp eax }
 #define VARJMP(a) { _asm jmp a }
-#elif defined(__GNUC__) || defined(__clang__)
-#define EAXJMP(a) { __asm__ volatile("mov eax, %0\n" "jmp eax" :: "i" (a)); }
-#define VARJMP(a) { __asm__ volatile("jmp %0" :: "m" (a)); }
+#define WRAPARG(a) ((int)a)
+#else
+#define EAXJMP(a) __asm__ volatile("mov eax, %0\n" "jmp eax" :: "i" (a));
+#define VARJMP(a) __asm__ volatile("jmp %0" :: "m" (a));
+#define WRAPARG(a)
```

Comment:
Weird, I thought you dealt with those already? Is this just a cleanup then?

Reply:
Clang is stricter with what can go in naked functions (and I didn't test Clang before the first MR was merged)